guides/common/modules/proc_adding-errata-to-an-incremental-content-view.adoc

```diff
@@ -4,7 +4,10 @@
 If errata are available but not installable, you can create an incremental Content View version to add the errata to your content hosts.
 For example, if the Content View is version 1.0, it becomes Content View version 1.1, and when you publish, it becomes Content View version 2.0.
 
-To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-errata-to-an-incremental-content-view[].
+IMPORTANT: If your content view version is old, you might encounter incompatibilities when incrementally adding enhanacement errata. 
```

Comment:
```suggestion
IMPORTANT: If your content view version is old, you might encounter incompatibilities when incrementally adding enhancement errata. 
```

Reply:
That's a good catch, this typo was fixed in another commit on master. So instead of applying your suggestion, I cherry-picked that whole commit: https://github.com/theforeman/foreman-documentation/pull/2505/commits/81e761139d41450d1629ad9c191918dff6884b4a